### DISCUSSION TITLE
Use `PNPM` instead of `Yarn`

### DISCUSSION THREAD

**Discussion by vedxp** [vuejs/vitepress#1431] [Q&A] [answered]
I want to use `PNPM` to build my site.

How can I use it instead of `Yarn`, since there's nothing documented about it?

**Comment by g4rry420** [comment, marked as answer]
@VedxP You can use any package manager to build your site - npm, yarn or pnpm. To use pnpm, go through their installation guide - https://pnpm.io/installation#using-npm and use pnpm instead of yarn - https://vitepress.vuejs.org/guide/getting-started

**Reply by vedxp** [reply]
Thanks for the answer.

I'll surely do the same!
